org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/SourceLayout.java

```diff
@@ -45,23 +45,23 @@ public String getOutputFor(Outlet outlet) {
 			case ROOT:
 				return "";
 			case MAIN_JAVA:
-				return "bin";
+				return "target/classes";
```

Comment:
is this for maven or for all cases?

Reply:
There are two cases:
- `SourceLayout.PLAIN`
- `SourceLayout.MAVEN`
But in fact the output folders are now aligned between both. In contrast to the source folders.

Reply:
See my mistake. Have confidered only two cases (Maven+Eclipse Layout, Maven+Maven Layout) and skipped the last one (No Maven+Eclipse Layout). Will have a look about the last case.

Reply:
you can have gradle with both layouts too